### DISCUSSION TITLE
Didn't get access to Github Copilot

### DISCUSSION THREAD

**Discussion by miumerfarooq** [community/community#6878] [Copilot Conversations] [answered]
Till now I didn't get access to Github Copilot. How will I get access to it?

**Comment by davecheney** [comment, marked as answer]
@Umer-Farooq10 did you sign up for the tech preview waitlist. If not, please register and we’ll get to you as we add capacity to the service.

**Reply by kre-10** [reply]
Is there heavy demand for copilot? I've been signed up for more than 3 months and sadly have yet to receive an invite.

**Reply by davecheney** [reply]
yes

**Reply by TiagoCerqueira** [reply]
same as kre-10, nothing and i'm still waiting for more 3 months

**Reply by NoamGoren** [reply]
I as well signed up more than 3 months ago, still waiting.

**Reply by xamelion** [reply]
Yes. I would very much like to try =)

**Reply by aymengraoui** [reply]
i've been signed up since the release and my friends signed up after me they got invitation and i didnt 😞

**Reply by makusu720** [reply]
same here, i signed up from the first day

**Reply by MohamedDev99** [reply]
I as well signed up more than 3 months ago, still waiting.  why?

**Reply by NeXuS4Developer** [reply]
i am waiting for 1 week.

**Reply by omeramiel** [reply]
I signed up 2 months ago, still waiting...

**Reply by VRER1997** [reply]
Me, too.

**Reply by YuvalKrispin** [reply]
+1 can't wait to get it...
I have a friend that got it after a week and I'm waiting for months...

**Comment by miumerfarooq** [comment]
I've been signed up for more than 3 months and sadly have yet to receive
an invite
